uwrt_mars_rover_blackfly_calibration/launch/blackfly_calibration.launch.xml

```diff
@@ -0,0 +1,9 @@
+<launch>
+
+ <arg name="topic" default="/blackfly/image/raw"/>
+ 
+ <node pkg="uwrt_mars_rover_blackfly_calibration" exec="broadcaster">
+   <param name="topic" value="$(var topic)"/>
+ </node>
+
```

Comment:
i had trouble starting the cameracalibrator node from this launch file. the cameracalibrator would complain about the ros-args flag which is automatically passed down. can i get some help with this next meeting after i re-add the code?